src/app/components/container-days/container-days.controller.js

```diff
@@ -0,0 +1,24 @@
+export default class ContainerDaysController {
+    constructor($state, openWeatherMapsService, weatherContainerService) {
+        this.$state = $state;
+        this.openWeatherMapsService = openWeatherMapsService;
+        this.weatherContainerService = weatherContainerService;
+        this.listDays = [];
+    }
+
+    $onInit() {
+        if (this.$state.params.city) {
+            this.param = this.$state.params.city.toLowerCase();
+        }
+        this.openWeatherMapsService.getCurrentWeather(this.param)
+        .then(response => {
+            this.listDays = this.weatherContainerService.getListToShow(response);
+        })
+        .catch(response => {
+            this.$state.go('errorInfo', {city: this.param, status: response.status});
+        });
+        return this.listDays;
```

Comment:
is not necessary to return anything on $onInit
also, please add some identation on .then and .catch methods